### ISSUE TITLE
Question on validation dataset for TSC

### ISSUE THREAD

**Issue by Wwwwei** [hfawaz/dl-4-tsc#51]
Hi, thanks for you good work.
I am new in TSC and try to follow your work. But I am confued about the validation dataset code in this repo.
It seems that some classifiers lack validation datasets and just use the test dataset instead, e.g., encoder and inception. 
In this way, how do I determine the best model to get the results of test datasets in TSC problem?

Thank you.

**Comment by hfawaz** [comment]
Hello thanks for your interest. 

I believe you can use a random split from the training set. 

Currently models do not use any validation set and their hyperparameters are fixed - which is suboptimal yes. 

Best of luck !

**Comment by Wwwwei** [comment]
> Hello thanks for your interest.
> 
> I believe you can use a random split from the training set.
> 
> Currently models do not use any validation set and their hyperparameters are fixed - which is suboptimal yes.
> 
> Best of luck !

Thanks for your reply. 
I notice that some datasets in UCR are too small to split into train dataset and validation dataset.
Any suggestions for this?

Thank you.

**Comment by hfawaz** [comment]
Maybe do a cross validation several splits/folds then

**Comment by Wwwwei** [comment]
Very helpful. Thanks.
